packages/itwinui-css/src/tree/tree.scss

```diff
@@ -41,37 +44,34 @@ $iui-expander-button-width: calc(var(--iui-component-height-small) + 2px);
 .iui-tree-node {
   display: flex;
   cursor: pointer;
-  padding-inline: var(--iui-size-xs);
+  padding-inline: var(--_iui-tree-node-padding-inline);
   align-items: center;
   border-radius: var(--iui-border-radius-1);
 
   &-checkbox {
-    margin-inline-end: var(--iui-size-xs);
+    margin-inline-end: var(--_iui-tree-node-padding-inline);
   }
 
   &-content {
     display: flex;
+    column-gap: var(--_iui-tree-node-padding-inline);
     align-items: center;
-    min-block-size: calc(var(--iui-size-s) * 3);
-    margin-inline-start: calc(#{$iui-expander-button-width} * (var(--level, 0)));
+    min-block-size: var(--_iui-tree-row-height);
+    margin-inline-start: calc(var(--_iui-expander-button-width) * 0.5 * var(--level, 0));
     overflow: hidden;
     padding-inline-start: var(--iui-size-3xs);
     inline-size: 100%;
 
     &-icon {
-      margin-inline: $iui-expander-inline-padding;
+      margin-inline: var(--_iui-tree-node-content-icon-margin-inline);
```

Comment:
this variable does not exist

Reply:
Hmm, must have been leftover from something I was trying?  I don't know, removed.